src/util/graph.creator.ts

```diff
@@ -84,12 +84,35 @@ export class GraphCreator {
         };
     }
 
-    private static resolveModule(rawImportLocation: string): string {
+    private static resolveModule(rawImportLocation: string, originDirectory: string): string {
         if (/[.]ts$/.test(rawImportLocation)) {
+            // import ends in .ts => has to be a file
             return rawImportLocation;
         }
+
+        if (
+            fs.existsSync(path.join(originDirectory, rawImportLocation, '.ts')) &&
+            fs.lstatSync(path.join(originDirectory, rawImportLocation, '.ts')).isFile()
+        ) {
+            return `${rawImportLocation}.ts`;
+        }
+
+        // Not a .ts file
+        if (
+            fs.existsSync(path.join(originDirectory, rawImportLocation)) &&
+            fs.lstatSync(path.join(originDirectory, rawImportLocation)).isDirectory()
+        ) {
+            if (
+                fs.existsSync(path.join(originDirectory, rawImportLocation, '/index.ts')) &&
```

Comment:
Uff, I don't know the details here either to be honest :/ But the coverage in the tests looks to be enough for do our power, and otherwise we'll notice soon enough :D

Reply:
I looked it up and path.join normalizes input. So it's fine!